indexers/keyvalue/RedisDBIndexer/tests/test_redisdbindexer.py

```diff
@@ -1,8 +1,10 @@
 from pathlib import Path
 
 import pytest
+import redis
```

Comment:
good change in here, but just fyi I meant not putting the import inside a `for loop`

Reply:
true, I got you. This was a helpful recommendation. I just thought since the imports are executed for sure anyways, I can import them directly at the top.